### PR TITLE
docs: Update portal README with manual workflow instructions

### PR DESCRIPTION
**Pull request by kyeah**
## 🎫 Ticket

N/A

## 🛠 Changes

- Add scripts to generate GOLDEN_MACAROON as part of `make portal-console` and `make portal-sh`
- Update README with instructions on how to go through Org setup and AO invitation flow
- Reorganize README a bit to consolidate test instructions and such

## ℹ️ Context for reviewers

Wanted to go through the current workflow and had to trawl through the code a bit to figure out AO invitations as it is currently built. Added documentation while getting it to work.

## ✅ Acceptance Validation

Walked through the steps as written in order to get AO invitation working for myself.

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

### CODE REVIEW

File: dpc-portal/bin/console [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
```
Comment (jdettmannnava) [inline dpc-portal/bin/console:1]: I don't think this should be necessary, as this happens when the docker container is initialized, so these variables should already be set?

File: dpc-portal/README.md [hunk 2]
```diff
@@ -1,54 +1,97 @@
...
+    ```
 
-## View Components
+## Manual Workflows
 
-We utilize the [ViewComponent](https://viewcomponent.org/) library to create custom components. These components are documented and viewable within [Lookbook](http://localhost:3100/portal/lookbook).
+To support manual workflows, the system offers rake commands that perform administrative commands for new accounts. These rake commands can be run within a portal shell (`make portal-sh`).
 
-Emails are not viewable in lookbook, but rather [here](http://localhost:3100/portal/rails/mailers/).
+-   `rails dpc:make_org` - Creates a new provider organization.
```
Comment (jdettmannnava) [inline dpc-portal/README.md:36]: I actually don't know if this is useful any more.

File: dpc-portal/bin/console (hunk 1 above)
Comment (kyeah) [reply dpc-portal/bin/console:1]: It's only run as part of the entrypoint script / as part of running the rails or sidekiq apps. When you run a separate interactive shell or rails console, it doesn't have the env var set there.

File: dpc-portal/README.md (hunk 2 above)
Comment (kyeah) [reply dpc-portal/README.md:36]: oops! correct — didn't look closely. the provider_organization record is created as part of invite_ao, maybe we can just remove make_org then.

File: dpc-portal/bin/console (hunk 1 above)
Comment (jdettmannnava) [reply dpc-portal/bin/console:1]: Ah, since I always run against a web container, I was confused

Comment (jdettmannnava) [reply dpc-portal/bin/console:1]: `docker exec -it dpc-app-dpc_portal-1 rails c`
which is actually in my .zshrc

Comment (jdettmannnava) [reply dpc-portal/bin/console:1]: Same with shell

Comment (jdettmannnava) [reply dpc-portal/bin/console:1]: So in order to adjust three files to any changes in initialization, maybe recommend running against a web container?

Comment (kyeah) [reply dpc-portal/bin/console:1]: Hmm our commands are functionally the same (the docker-compose command in the Makefile just uses the container name defined in the docker-compose.portals.yml file.)

```
$ docker exec -it dpc-app-dpc_portal-1 rails c
irb(main):002> ENV['GOLDEN_MACAROON']
=> ""
```

I also confirmed by clicking through the UI that GOLDEN_MACAROON is set within the rails app and can interact with the DPC API.

Do you have GOLDEN_MACAROON set in your shell / zshrc? 🤔

File: dpc-portal/README.md [hunk 3]
```diff
@@ -1,54 +1,88 @@
...
 
-# Running via Docker
+# Local Development
+
+## Running via Docker
 
 The DPC website can be run locally via docker. Follow the below steps to build and run the website into a Docker container.
 
 -   Run `make portal` to build the docker image, and `make start-portal` to start dpc-portal in a docker container.
+-   The DPC API is required for much of the Portal's functionality. To run the DPC API, run `make start-api`.
```
Comment (jdettmannnava) [inline dpc-portal/README.md:12]: Doesn't this need to go first to make the golden macaroon?

File: dpc-portal/README.md [hunk 4]
```diff
@@ -1,54 +1,88 @@
...
+    $ rails dpc:invite_ao INVITE=Bob,Hoskins,bob@example.com,7838426501
 
-Read more about the assets pipeline [here](/docs/portal/assets-pipeline.md).
+    Invitation created for Bob Hoskins for Organization 7838426501
+    http://localhost:3100/portal/organizations/2/invitations/4/accept
+    ```
+
+## Local AO Verification Scenarios
+
+One of the following fake SSNs are required on your Login.gov sandbox account in order to successfully pass AO-Organization verification:
```
Comment (jdettmannnava) [inline dpc-portal/README.md:48]: Maybe add:
'If you need to use another ssn, you can add it to the [ao_ssns variable ](https://github.com/CMSgov/dpc-app/blob/65c5e7b4adab82ae75aa4005dbb6a4ebb199a86d/dpc-portal/spec/support/fake_cpi_gateway.rb#L76) in the enrollment roles endpoint'

File: dpc-portal/README.md [hunk 5]
```diff
@@ -1,54 +1,88 @@
...
+You can check the status of any jobs by going to the [Sidekiq dashboard](http://localhost:3100/portal/sidekiq).
 
-Then run `make start-portal`. You can confirm that this worked by opening the Rails console and checking `ENV.fetch('GOLDEN_MACAROON')`.
+## Accessing the CPI API Gateway Locally
 
-You can check the status of any jobs by going to the Sidekiq dashboard at `http://localhost:3100/portal/sidekiq`
+By default, the local portal is connected to a mock CPI API Gateway server with preset responses. To connect with the VAL CPI API Gateway environment, you'll need to be running Zscaler. To connect the local portal container over Zscaler, you'll want to add the following directly before line 5 of the Dockerfile and rerun the build steps:
+
+`COPY Zscaler-Root-CA.pem /usr/local/share/ca-certificates/ZScaler-Root-CA.pem`
+
```
Comment (jdettmannnava) [inline dpc-portal/README.md:85]: Maybe also add:
You will also need to set the following environment variables in docker-compose.portals.yml
CPI_API_GW_BASE_URL=https://val.cpiapi.cms.gov/
CMS_IDM_OAUTH_URL=https://impl.idp.idm.cms.gov/

**Review by jdettmannnava — APPROVED** [review]
LGTM!

**Pull request merged**